lib/Logger/Logger.h

```diff
@@ -68,11 +66,7 @@ class Logger {
          * @param line The line number of the log message
          * @param logmsg Log message to be sent as payload
          */
-        void log(const Level level,
-                 const String& file,
-                 const __FlashStringHelper* function,
-                 uint32_t line,
-                 const char* logmsg);
+        void log(const Level level, const String& file, const __FlashStringHelper* function, uint32_t line, const char* logmsg);
```

Comment:
i thought we decided to keep the version with new lines after ","

Reply:
That only happens when the line becomes too long. I think we can also switch it *always* on if you wish, but I'm not sure that will help. @fiendie wanted longer lines, if we break at every comma, we will hardly surpass the 20 characters per line :D

Reply:
I also think it's nicer to only break when the line becomes too long